File: src/main/java/com/wanted/preonboarding/ticket/domain/dto/request/ReservationCancelRequest.java
package com.wanted.preonboarding.ticket.domain.dto.request;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class ReservationCancelRequest {
    private Integer reservationId;

    protected ReservationCancelRequest() {
    }

    public ReservationCancelRequest(Integer reservationId) {
        this.reservationId = reservationId;
    }
}

File: src/main/kotlin/com/wanted/preonboarding/ticket/presentation/request/ReservationCancelRequest.kt
package com.wanted.preonboarding.ticket.presentation.request

import com.wanted.preonboarding.ticket.domain.SeatInfo
import com.wanted.preonboarding.ticket.domain.UserInfo
import java.util.UUID

data class ReservationCancelRequest(
    val performanceId: UUID,
    val seatInfo: SeatInfo,
    val userInfo: UserInfo,
)
